engine.py

```diff
@@ -504,9 +504,48 @@ def save_to_path(self, document, path):
 
             # make the document being processed the active document
             self.adobe.app.activeDocument = document
-
-            # TODO need to use appropriate save options here (jpg, tiff, etc)
-            document.saveAs(self.adobe.File(path))
+        
+            (_, ext) = os.path.splitext(path)
+            ext = ext.lower()
+
+            # these extensions are supported by photoshop
+            # PICTResourceSaveOptions is skipped for now, need a way to differentiate PICT
+            # files from PICT resource files
+            # DCS1_SaveOptions is not used for ".dcs" files, DCS2_SaveOptions is used instead
+            if ext ==".bmp":
+                save_options = self.adobe.BMPSaveOptions
+            elif ext ==".dcs":
+                save_options = self.adobe.DCS2_SaveOptions
+            elif ext ==".eps":
+                save_options = self.adobe.EPSSaveOptions
+            elif ext ==".gif":
+                save_options = self.adobe.GIFSaveOptions
+            elif ext in [".jpg", ".jpeg"]:
+                save_options = self.adobe.JPEGSaveOptions;
+                save_options.quality = 12
```

Comment:
Suggest a comment saying why we're setting this to something other than the default.